docs/code/creating-and-writing-nwbfile-2.py

```diff
@@ -0,0 +1,32 @@
+
+def main():
+    import os.path
+
+    # example: start
+    from datetime import datetime
+
+    from pynwb import NWBFile, TimeSeries, get_manager
+    from pynwb.form.backends.hdf5 import HDF5IO
```

Comment:
nitpick: Should the imports be at the top of the file?

Reply:
Since we want have them showing up in the example, they have been purposely added in the middle .. just before each comment of the form `# example: start` / `# example: end` that are used in the sphinx directive `litteralinclude`.